controllers/apirule_controller.go

```diff
@@ -148,6 +201,8 @@ func (r *APIRuleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 func (r *APIRuleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&gatewayv1beta1.APIRule{}).
+		Owns(&corev1.ConfigMap{}).
+		WithEventFilter(&configMapPredicate{Log: r.Log}).
```

Comment:
This seems to be preventing the controller from reconciling on apirule creation, as this is applied to all watched resources
```suggestion
```

Reply:
Fixed that, but still don't get reconciles for CM updates, looking into it.